bin/commands/plugins.js

```diff
@@ -427,27 +443,29 @@ function checkOptions(plugin, options) {
     process.exit(1);
   }
 
-  // --install is the only option working without specifying a plugin name
-  if (!plugin && !options.install) {
-    console.error(clcError('A plugin [name] is required for this operation'));
-    process.exit(1);
-  }
+  // --install and --list are the only options working without specifying a plugin name
+  if (!options.list) {
```

Comment:
This code should be improved : 
* --install and --list are mutually exclusive (what is not checked yet)
* the condition "if (!plugin && !options.install && !options.list)" should not be nested.

Improvement proposal : 
```
if (!plugin && !options.install && !options.list) {
  console.error(clcError('A plugin [name] is required for this operation'));
  process.exit(1);
}

if (options.install && options.list) {
  console.error(clcError('Options --install and --list are mutually exclusive));
  process.exit(1);
}

// then put the rest of the code, unchanged
```